api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -115,6 +124,9 @@ public void setUp() {
             surveyModuleDao,
             cohortBuilderMapper,
             mySQLStopWordsProvider);
+
+    workbenchConfig = WorkbenchConfig.createEmptyConfig();
```

Comment:
Not needed.

Reply:
Will remove these since the test is no longer there...